test/RestrictedNORI.t.sol

```diff
@@ -12,3 +14,27 @@ contract RestrictedNORI_initialize is UpgradableRestrictedNORIMock {
     // todo rest
   }
 }
+
+contract RestrictedNORI__linearReleaseAmountAvailable is
+  NonUpgradableRestrictedNORIMock
+{
+  using RestrictedNORIUtils for Schedule;
```

Comment:
I'd be surprised if this is required given that NonupgradeableRestrictedNORIMock inherits from RestrictedNORI